enhancements/update/configurable-clusterversion-signature-stores.md

```diff
@@ -0,0 +1,194 @@
+---
+title: configurable-clusterversion-signature-stores
+authors:
+  - "@PratikMahajan"
+reviewers:
+  - "@LalatenduMohanty"
+  - "@wking"
+approvers:
+  - "@LalatenduMohanty"
+api-approvers:
+  - "@deads2k"
+creation-date: 2023-09-27
+last-updated: 2023-10-31
+tracking-link:
+  - https://issues.redhat.com/browse/OTA-916
+---
+
+# ClusterVersion has option to replace default signature stores
+
+## Release Signoff Checklist
+
+- [X] Enhancement is `implementable`
+- [ ] Design details are appropriately documented from clear requirements
+- [ ] Test plan is defined
+- [ ] Graduation criteria for dev preview, tech preview, GA
+- [ ] Operational readiness criteria is defined
+- [ ] User-facing documentation is created in [openshift-docs](https://github.com/openshift/openshift-docs/)
+
+## Summary
+
+CVO currently fetches the upstream urls for signature verification through [cluster-update-keys]. 
+To increase flexibility and improve disconnected cluster user experience, we need to provide a way for the CVO to 
+get the required signatures from custom signature stores or the OSUS running in the disconnected environment.
```

Comment:
Given the use case, it seems to me like the users may want to be able to disable the regular stores, since they know they won't be routable. I assume in disconnected environments today the CVO just spews a bunch of errors when it tries to fetch?